packages/workbox-webpack-plugin/src/lib/get-script-files-for-chunks.js

```diff
@@ -17,7 +19,10 @@ module.exports = (compilation, chunkNames) => {
     const chunk = chunks.find((chunk) => chunk.names.includes(chunkName));
     if (chunk) {
       for (const file of chunk.files) {
-        scriptFiles.add(resolveWebpackURL(publicPath, file));
+        // See https://github.com/GoogleChrome/workbox/issues/2161
+        if (upath.extname(file) === '.js') {
```

Comment:
I doubt non-js extensions ever happen with webpack chunks, but I wonder if we want to be a bit more forgiving here and accept things like `.mjs`. What do you think?

Reply:
See my comment in the PR description 😄 

> (importScripts() will never play nicely with ES modules, so `.mjs` is a non-issue.)

Reply:
Gah, sorry, missed that!